src/status_im/contexts/wallet/collectible/utils.cljs

```diff
@@ -69,3 +67,14 @@
         contract-address (-> id :contract-id :address)
         token-id         (-> id :token-id)]
     (str chain-id contract-address token-id)))
+
+(defn remove-duplicates-in-ownership
+  [ownership]
+  (->> ownership
+       (reduce (fn [acc {:keys [address timestamp] :as owner}]
+                 (let [existing-owner (get acc address)]
+                   (if (or (nil? existing-owner) (> timestamp (:timestamp existing-owner)))
+                     (assoc acc address owner)
+                     acc)))
+               {})
+       vals))
```

Comment:
Sometimes, the status-go returns duplicate data in the list, we do a cleanup based on the address and timestamp of the balance.